src/screens/settings-screen.tsx

```diff
@@ -5,6 +5,8 @@ import {
 	StyleSheet,
 	Text,
 	View,
+	TouchableOpacity,
+	Clipboard,
 } from "react-native";
```

Comment:
Editor suggest:

Clipboard has been extracted from react-native core and will be removed in a future release. It can now be installed and imported from @react-native-community/clipboard instead of 'react-native'.

https://github.com/react-native-clipboard/clipboard